docs/dictionary/property/styledText.lcdoc

```diff
@@ -102,11 +102,12 @@ encoding.
 
 For example, take the following content consisting of two paragraphs:
 
-Centered Hello World
+<p style="text-align: center">Centered **Hello** World</p>
 
-Left-aligned Hello unicodeString
+Left-aligned <span style="color: rgb(255,0,0)">Hello</span> unicodeString
 
```

Comment:
I do not think the 2 changes above are needed. The original lines:
```
Centered Hello World

Left-aligned Hello unicodeString
```
just describe in words how the content looks like.